src/Agent/NewRelic.Api.Agent/NewRelic.cs

```diff
@@ -586,7 +719,14 @@ public static void DisableBrowserMonitoring(bool overrideManual = false)
         [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
         public static void SetUserParameters(string? userName, string? accountName, string? productName)
         {
-            System.Diagnostics.Trace.WriteLine("NewRelic.SetUserParameters(String userName, String accountName, String productName)");
+            try
+            {
+                System.Diagnostics.Trace.WriteLine("NewRelic.SetUserParameters(String userName, String accountName, String productName)");
```

Comment:
Wouldn't this fail the compile since the params aren't actually being referenced?

Reply:
We should do `nameof(paramName)` on all params of all methods regardless of whether they are nullable.